raster-foundry/raster-foundry
# Bump STAC version in exports to 1.0.0

## Overview

This PR bumps the STAC version in exports to 1.0.0. It also resolves resulting dependency lint (lots of upgrades). It also adds a nice README to the resulting exports, after some discussion with @jamesmcclain about how exports from different eras are difficult to figure out how to use if you're bouncing back and forth. It _also_ wound up removing the `backsplash-export/` subproject, since we don't do that anymore and dep upgrades were hard (poor `maml` could really use some love).

### Checklist


### Demo

README posted in [slack](https://azavea.slack.com/archives/CDB8M6W7M/p1628198257019600)

### Notes

~There's enough going on here that I'd quite like to have this in before QA tomorrow, since akka-http, circe, sttp, and some other fun binary compatibility buddies that are core to everything we do were upgraded quite a distance.~

Well this turned into a lot of fun. Most of the core functionality has been done for a bit, but getting to binary compatibility in the api integration test took a while. I couldn't make sttp / circe / friends agree, so I eventually ripped out sttp as the client and used the http4s blaze client instead. That worked pretty nicely, but the cost is that the integration test runs much more slowly now. We could consider separating that into a different test suite in GHA or something, especially since we're already interested in evaluating other CI options if we're going to turn on scala-steward.

## Testing Instructions

- bring up servers after assembly
- point GroundWork to local
- draw some labels
- create an export
- fetch your export from S3 and check out the README
- open it in something that knows how to preview markdown and confirm that it's a beautiful that tells you more than you've ever known about your exports
- browse around and see if you can get to any super broken interactions

Closes azavea/raster-foundry-platform#1285

Closes #5462
